File: backendbuddyfinv/src/main/java/com/es/backendbuddyfinv/dto/IngresoDTO.java
package com.es.backendbuddyfinv.dto;

import java.time.LocalDate;
import lombok.Data;

@Data
public class IngresoDTO {
    private Long idIngreso;
    private LocalDate fecha;
    private double totalDiario;
    private int totalFacturas;
    private String nombreEmpleado;
}

File: frontendbuddyfinv/src/models/IngresoDTO.js
export class IngresoDTO {
    constructor({
      idIngreso = null,
      fecha = null,
      totalDiario = 0,
      totalFacturas = 0,
      nombreEmpleado = ''
    } = {}) {
      this.idIngreso = idIngreso;
      this.fecha = fecha;
      this.totalDiario = totalDiario;
      this.totalFacturas = totalFacturas;
      this.nombreEmpleado = nombreEmpleado;
    }
  }